draw_flip_book/lib/main.dart

```diff
@@ -182,19 +243,27 @@ class _FlipBookPageState extends State<FlipBookPage> {
 
   void _toggleFramesVisibility() {
     if (_replayFrames) {
-      if (currentFrame == 1) {
-        currentFrame = 3;
-        _isVisible3 = true;
-        _replayFrames = false;
+      if (currentFrame == 4) {
```

Comment:
Should we be using 0-indexed frames to prevent confusion?

Reply:
makes sense, changed to be 0 based